src/explore-education-statistics-common/src/hooks/useClickAway.ts

```diff
@@ -17,7 +17,7 @@ export default function useClickAway<
   events: (keyof DocumentEventMap)[] = ['click'],
 ) {
   useEffect(() => {
-    const handler: EventListenerOrEventListenerObject = event => {
+    const handler: EventListenerOrEventListenerObject = (event: Event) => {
```

Comment:
This isn't related to the worker change, but `tsc` started complaining about it 🤷